src/fork_gc.c

```diff
@@ -513,12 +513,12 @@ static void FGC_childCollectTags(ForkGC *gc, RedisSearchCtx *sctx) {
 }
 
 static void FGC_childScanIndexes(ForkGC *gc, IndexSpec *spec) {
-  RedisSearchCtx sctx = SEARCH_CTX_STATIC(gc->ctx, spec);
-
-  FGC_childCollectTerms(gc, &sctx);
-  FGC_childCollectNumeric(gc, &sctx);
-  FGC_childCollectTags(gc, &sctx);
-
+	RedisSearchCtx sctx = SEARCH_CTX_STATIC(gc->ctx, spec);
```

Comment:
let's keep the indentations to 2 spaces per level

Reply:
ok